proposals/p2187.md

```diff
@@ -0,0 +1,36 @@
+# Update sum types design
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/2187)
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Abstract](#abstract)
+-   [Problem](#problem)
+-   [Proposal](#proposal)
+
+<!-- tocstop -->
+
+## Abstract
+
+This proposal updates the design of [p0157](p0157.md) to reflect subsequent
```

Comment:
```suggestion
This proposal updates the design of [#157](p0157.md) to reflect subsequent
```
We don't seem to use `pNNNN` in text to name Carbon proposals; the design style guide suggests using `#NNNN` instead.